.github/workflows/dry-run.yml

```diff
@@ -103,6 +103,27 @@ jobs:
           skip_tests: true
           download_dependencies: true
 
+  buildBoardComponents:
+    name: Build board components
+    runs-on: ubuntu-latest
+    needs:
+      - buildGlobalStyles
+      - buildBrowserTestTools
+      - buildDocumenter
+      - buildTestUtils
+      - buildComponentToolkit
+      - buildComponents
+    steps:
+      - name: Download component artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: components-package
+      - uses: cloudscape-design/.github/.github/actions/build-package@main
+        with:
+          package: board-components
+          skip_tests: true
```

Comment:
why are we skipping tests here?

Reply:
Right. Because components has separate unit, integ, and a11y test workflows that run in parallel, e.g. https://github.com/cloudscape-design/.github/blob/main/.github/workflows/dry-run.yml#L106. Other packages just rely on running `npm test`

Reply:
Fixed in the next revision